osi_object.proto

```diff
@@ -54,6 +54,15 @@ message Vehicle
     /// The id of the driver of the (ego) vehicle.
     /// Field need not be set if ego_vehicle is set to false.
     optional Identifier driver = 10;
+    
+    /// The vector pointing from the bounding box center point (base.position) to the middle (in x, y and z) 
+    /// of the rear axle under neutral load conditions.
+    optional Vector3d bbcenter_to_rear = 11;
+
+	/// The vector pointing from the bounding box center point (base.position) to the middle (in x, y and z) 
+	/// of the front axle under neutral load conditions.    		
+	optional Vector3d bbcenter_to_front = 12;
+    
 
```

Comment:
What do other people think? Would replacing relative_ego_origin disrupt some OSI logic?
@nHirsenkorn @TimoHankeBMW Your input would be appreciated.
If you agree I could make the necessary changes.